docs/RELEASE_NOTES.md

```diff
@@ -7,12 +7,13 @@
   - Reduce the number of permissions for operator role.
   - Support SHA-1 as an authentication method.
   - Upgraded `mongodbcommunity.mongodbcommunity.mongodb.com` CRD to `v1` from `v1beta1`
-    *  Users upgrading their CRD from v1beta1 to v1 need to set: `spec.preserveUnknownFields` to `false` in the CRD file `config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml` before applying the CRD to the cluster.
+    - Users upgrading their CRD from v1beta1 to v1 need to set: `spec.preserveUnknownFields` to `false` in the CRD file `config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml` before applying the CRD to the cluster.
   - Made service name configurable in mongdb custom resource with statefulSet.spec.serviceName
+  - Add Kube-linter as a github actions.
```

Comment:
I don't think we ever specified github actions as part of the release notes. I think we focus on more user-facing changes